assets/src/infrastructure/ui/layout/menu/Menu.tsx

```diff
@@ -4,8 +4,12 @@ import { Menu as ChakraMenu } from '@chakra-ui/core';
 
 import { MenuProps } from './types';
 
-const Menu: React.FC<MenuProps> = ({ children, ...props }) => {
-	return <ChakraMenu {...props}>{children}</ChakraMenu>;
+const Menu: React.FC<MenuProps> = ({ closeOnBlur = true, children, ...props }) => {
```

Comment:
set `closeOnBlur` explicitly to `true`, but it's still not closing when it's clicked outside of the area

Reply:
@manzoorwanijk I had reverted this

Reply:
I moved the tooltip out of the dropdown menu and then wrapped that component in the `witTooltip` HOC:

https://github.com/eventespresso/event-espresso-core/pull/2838/commits/c5e220e23329a6160d484282c88897089125ad0e#diff-1b4e6999f94ed2e1742f9f0aff62127eR8-R33

so that might have caused this change, but we should jump back to a commit before the above was merged to confirm